lib/releases.js

```diff
@@ -144,7 +144,8 @@ const generateChangeLog = (mergedPullRequests, config) => {
         template(config['change-template'], {
           $TITLE: pullRequest.title,
           $NUMBER: pullRequest.number,
-          $AUTHOR: pullRequest.author ? pullRequest.author.login : 'ghost'
+          $AUTHOR: pullRequest.author ? pullRequest.author.login : 'ghost',
+          $DESC: pullRequest.body
```

Comment:
and this, is the only required change for the `$DESC` being available

Reply:
Thanks @jetersen 
what's the process of making a github action so that I can use the updated code in my private repo?